internal/controller/operatorconfigmap_controller.go

```diff
@@ -500,6 +500,64 @@ func (c *OperatorConfigMapReconciler) shouldEnableDriver(driverKey string) bool
 	return enableDriver
 }
 
+func (c *OperatorConfigMapReconciler) getTopologyLabels(storageClients *v1alpha1.StorageClientList) map[string]struct{} {
```

Comment:
Let's loop over the storageClient's first and merge the topology from the storageClient status first, similar to what was done before.
If topologyDomainLabelsSet is empty, then use the topology specified in the configmap

Reply:
Please add a comment on the function which takes the highest precedence

Reply:
added